certification/results.go

```diff
@@ -0,0 +1,25 @@
+package certification
+
+import (
+	"time"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/check"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/runtime"
+)
+
+type openshiftClusterVersion = runtime.OpenshiftClusterVersion
```

Comment:
Feel free to ignore - does this still need to be here? It just looks like an alias, but I'm not sure why we would have used it.

Reply:
It's an alias that reaches into internal so it can still be used as a type here, without having to duplicate it, and so that it isn't exposed as an actual part of the API.